pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -850,8 +867,30 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to authenticate to OpenStack")
 	}
+	provider, err := openstack.NewClient(opts.IdentityEndpoint)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to authenticate to OpenStack")
+	}
+
+	// We need to fetch user-provided OpenStack cloud CA certificate and make gophercloud use it.
+	// Also it'll get injected into a ConfigMap mounted into kuryr-controller later on.
+	userCACert, err := getUserCACert(kubeClient)
```

Comment:
do you need here : if err != nil?

Reply:
Not necessarily, the idea is that when error happens this just proceeds as the cert was not set. And indeed that's the case as the ConfigMap will not exist when user had not set the certificates in install-config.yaml.

Reply:
awesome! thanks!